exercises/G-conditionals-2/exercise-3.js

```diff
@@ -8,7 +8,24 @@
   - Otherwise the grade is "F"
 */
 
-function calculateGrade(mark) {}
+function calculateGrade(mark) {
+  if (mark >= 80)
+  {
+    return "A";
+  }
+  else if (mark < 80 && mark > 60)
```

Comment:
Good. A better way to write it though is to write the else if statement in such a way where we check if it is greater than the lower value and less than the higher value:
else if (mark >60 && mark <80)